content/ko/docs/concepts/services-networking/_index.md

```diff
@@ -5,8 +5,50 @@ description: >
   쿠버네티스의 네트워킹에 대한 개념과 리소스에 대해 설명한다.
 ---
 
+## 쿠버네티스 네트워크 모델
```

Comment:
Thank you for your translation but it would have been better if you reflect original translator's artifacts. My saying is just based on my observation that you directly translated. See https://github.com/kubernetes/website/issues/32053 for more details (I have just closed this issue). Thank you

Reply:
네 확인하였습니다. 제가 #31695 와 #31698 (이 PR) 을 모두 담당했음에도
해당 섹션이 한 파일에서 다른 파일로 옮겨진 것을 인지하지 못하고
이전 파일에서는 기존 번역을 remove / 새 파일에서는 제가 원문을 보고 새로 번역을 하였습니다. 😢

말씀하신 내용 중에서 artifacts 가 무엇을 의미하는지 파악을 하지 못하여..
1) 원래 번역(#20408)의 저자이신 [pjhwa](https://github.com/pjhwa) 님을 이 commit의 co-author로 추가하는 것과
2) 원래 번역을 여기로 옮긴 뒤에 필요한 부분을 업데이트

하는 것 중에서 어떤 쪽을 제안해 주셨는지 알 수 있을까요? 😊

Reply:
I am now okay with your PR but want to highlight two things: 1) this is another unexpected issue resulted from spliting into multiple outdated update PRs 2) the translation style from you and @pjhwa are a little bit different so ideally it is a good idea to preserve the original translator's name on commit history. If you can propose a separate PR reflecting from pjhwa's translation style as co-author and he agrees with it, that would be perfect. For this PR, focusing outdated files is also a great idea.